lib/libyui_client/widgets/selectionbox.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Widgets
+    # Class representing a selectionbox in the UI. It can be YSelectionBox.
+    class Selectionbox < Widgets::Base
+      # Returns the list of items available to select from selection box.
+      # @return [Array] array of String objects.
+      # @example Get items from selection box with id "test_id"
+      #   {
+      #     "class" : "YSelectionBox",
+      #     "debug_label" : "selection box title",
+      #     "hstretch" : true,
+      #     "icon_base_path" : "",
+      #     "id" : "test_id",
+      #     "items" :
+      #     [
+      #       {
+      #         "label" : "selection 1",
+      #         "selected" : true
+      #       },
+      #       {
+      #         "label" : "selection 2"
+      #       },
+      #       {
+      #         "label" : "selection 3"
+      #       }
+      #     ],
+      #     "items_count" : 3,
+      #     "label" : "&selection box title",
+      #     "vstretch" : true
+      #   }
+      #
+      # @example
+      #   app.selectionbox(id: 'test_id').items
+      #   # selection 1
+      #   # selection 2
+      #   # selection 3
+      def items
+        property(:items).map { |x| x[:label] }
+      end
+
+      # Sends action to click the selection in UI.
+      # @param item [String] value to select from items.
+      # @return [selection] in case action is successful
```

Comment:
returns the object itself [Selectionbox]

Reply:
Done